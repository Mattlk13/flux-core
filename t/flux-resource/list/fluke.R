{"version": 1, "execution": {"R_lite": [{"rank": "0-100", "children": {"core": "0-3"}}], "starttime": 0.0, "expiration": 0.0, "nodelist": ["fluke[3-103]"], "properties": {"batch": "0-92", "debug": "93-100"}}}
