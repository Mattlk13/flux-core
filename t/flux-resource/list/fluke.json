{
  "all": {
    "version": 1,
    "execution": {
      "R_lite": [
        {
          "rank": "0-100",
          "children": {
            "core": "0-3"
          }
        }
      ],
      "starttime": 0.0,
      "expiration": 0.0,
      "nodelist": [
        "fluke[3-103]"
      ],
      "properties": {
        "debug": "93-100",
        "batch": "0-92"
      }
    }
  },
  "down": {
    "version": 1,
    "execution": {
      "R_lite": [
        {
          "rank": "14-15,21,40,58,62,73-74,76,89,94-95,100",
          "children": {
            "core": "0-3"
          }
        }
      ],
      "starttime": 0.0,
      "expiration": 0.0,
      "nodelist": [
        "fluke[17-18,24,43,61,65,76-77,79,92,97-98,103]"
      ],
      "properties": {
        "debug": "94-95,100",
        "batch": "14-15,21,40,58,62,73-74,76,89"
      }
    }
  },
  "allocated": {
    "version": 1,
    "execution": {
      "R_lite": [],
      "starttime": 0.0,
      "expiration": 0.0,
      "nodelist": [
        ""
      ]
    }
  }
}
