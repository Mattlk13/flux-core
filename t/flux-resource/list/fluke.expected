     STATE PROPERTIES NNODES   NCORES    NGPUS
      free batch          83      332        0
      free debug           5       20        0
 allocated                 0        0        0
      down batch          10       40        0
      down debug           3       12        0
